src/app/routes/applications/applications.component.ts

```diff
@@ -65,6 +65,13 @@ export class ApplicationsComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   openNewOrgComponent(): void {
+    if (!this.isIamEwcOwner) {
+      const namespace = 'orgowner.roles.orgcreator.apps.energyweb.iam.ewc';
+      const roleName = 'orgowner';
+      const siteUrl = `${location.origin}/#/enrol?org=${namespace}&roleName=${roleName}`;
+      window.open('//' + siteUrl, '_blank');
```

Comment:
Instead of `window.open`, could maybe just route to enrolment page. This is what is done when the `enrol` button is clicked from search screen. @VOSpdUkraine please adjust based on feedback from Olesia and John M in story. IMO, it would be better to just navigate to enrolment screen rather than new window but interested in your, Olesia and John M's opinions.
![image](https://user-images.githubusercontent.com/4407464/123122272-1bb88500-d403-11eb-9b5f-81c0a060fa7f.png)